nomad/csi_endpoint_test.go

```diff
@@ -19,7 +19,6 @@ import (
 	cconfig "github.com/hashicorp/nomad/client/config"
 	cstructs "github.com/hashicorp/nomad/client/structs"
 	"github.com/hashicorp/nomad/helper/uuid"
-	"github.com/hashicorp/nomad/lib/lang"
```

Comment:
This package doesn't exist in Nomad 1.4, so I've hacked together a reasonable equivalent here with an anonymous struct.